src/main/java/servlets/LoginServlet.java

```diff
@@ -36,11 +34,8 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType(CONTENT_TYPE_TEXT_HTML);
-    PrintWriter out = response.getWriter();
-    if (UserAuthUtil.isUserLoggedIn()) {
-      out.println(UserAuthUtil.getLogoutURL(REDIRECT_LINK));
-    } else {
-      // Send user to OAuth consent page.
+    response.getWriter().println(ResponseBuilder.toJson(UserAuthUtil.isUserLoggedIn(), REDIRECT_LINK));
+    if (!UserAuthUtil.isUserLoggedIn()) {
       response.sendRedirect(getOAuthRedirectURL());
```

Comment:
I though we would redirect to the OAuth endpoint after the first login page.

e.g the redirect link passed to the response builder should be based on if the user is logged in or not.

Reply:
#25